friendshipper/src-tauri/src/repo/operations/gh/submit.rs

```diff
@@ -473,6 +473,14 @@ impl SubmitOp {
             }
         }
 
+        // cleanup worktree branch
+        _ = git_client_worktree
```

Comment:
til you can do this without `let`

Reply:
I didn't even think about it, tbh. You do it this way in zig, which I'm guessing cribbed from rust :P